envoy.hcl

```
@@ -3,7 +3,7 @@ test = "envoy --version"
 strip = 2
 binaries = ["envoy"]
 
-// Artefacts are from the Tetrate Labs manifest:
+// Artifacts are from the Tetrate Labs manifest:
```

Believe it or not, "artefacts" is how it is spelt in Australia. Please revert.

---

sure thing. odd I never learned this, yet!

```suggestion
// Artefacts are from the Tetrate Labs manifest:
```